index.js

```diff
@@ -168,19 +168,28 @@ module.exports = function (buf) {
 		};
 	}
 
-	// needs to be before the `webm` check
-	if (buf[31] === 0x6D && buf[32] === 0x61 && buf[33] === 0x74 && buf[34] === 0x72 && buf[35] === 0x6f && buf[36] === 0x73 && buf[37] === 0x6B && buf[38] === 0x61) {
-		return {
-			ext: 'mkv',
-			mime: 'video/x-matroska'
-		};
-	}
-
+	// https://github.com/threatstack/libmagic/blob/master/magic/Magdir/matroska
 	if (buf[0] === 0x1A && buf[1] === 0x45 && buf[2] === 0xDF && buf[3] === 0xA3) {
-		return {
-			ext: 'webm',
-			mime: 'video/webm'
-		};
+		const sliced = buf.subarray(4, 4 + 4096);
```

Comment:
This is a TypedArray method, not a Buffer one. While Buffers inherit from TypedArray, they're not entirely the same and depending on something like this feels unsafe. I would just use `buf.slice()`, which actually does the same thing as `TypedArray#subarray()`.